plutus-core/plutus-core/src/PlutusCore/ASTSize.hs

```diff
@@ -0,0 +1,66 @@
+module PlutusCore.ASTSize
```

Comment:
Since this module is new I suggest to use:
- 2 space indentation, as per the CODESTYLE.adoc
- Explicit imports.

Reply:
About the 2space I agree, but about the explicit imports I kind of disagree.

I used to be back in the days also a proponent of explicit imports, but thanks to mpj I learned that managing and constantly editing the import list is a burden, even with hls helping you out (when it works).

Reply:
> I learned that managing and constantly editing the import list is a burden

I agree that its a burden for a writer, but its a relief for a reader.  Wildcard import shift the effort away from writers to readers. Since code is read more times that its written, I'd argue that we should optimise for reading.